Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/11-blue-green.md

```diff
@@ -0,0 +1,268 @@
+- Public facing hostname, the one used by the end users of the workloads/apps hosted into the clusters
+- blue cluster hostname, that is dedicated fot the internal validation
+- green cluster hostname, that is dedicated fot the internal validation
+
+The blue/green deployment can be summarized in 5 steps:
+1. T0: Blue Cluster is On, this means:
+  - "blue cluster" and "blue app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - "green cluster" and "green app gateway" with aks_turn_on=false and appgw_turn_on=false
+  - A record mapped with the PIP of the Blue Application Gateway
+  This is the default scenario in which you land following the [Getting Started with the default values](../README.md)
+2. T1: Green Cluster Deployment
+  - "blue cluster" and "blue app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - "green cluster" and "green app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - A record mapped with the PIP of the Blue Application Gateway
+3. T2: Sync K8S State between Blue and Green clusters
+4. T3: Traffic Switch to the green cluster
+  - "blue cluster" and "blue app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - "green cluster" and "green app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - A record mapped with the PIP of the Green Application Gateway
+5. T4: Blue cluster is destroyed
+  - "blue cluster" and "blue app gateway" with aks_turn_on=false and appgw_turn_on=false
+  - "green cluster" and "green app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - A record mapped with the PIP of the Green Application Gateway
+
+## T0: Blue Cluster is On
+
+  Follow the guide [Getting Started with the default values](../README.md). Where the default values are:
+
+- in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\05-Network-LZ\app-gateway.tf"
+```
+locals {
+  appgws = {
+    "appgw_blue" = {
+      name_prefix="blue"
+      appgw_turn_on=true
+    },
+    "appgw_green" = {
+      name_prefix="green"
+      appgw_turn_on=false
+    }
+  }
+}
+
+```
+
+- in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\07-AKS-cluster\aks-cluster.tf"
+
+```
+locals {
+  aks_clusters = {
+    "aks_blue" = {
+      name_prefix="blue"
+      aks_turn_on=true
+      k8s_version="1.23.5"
+      appgw_name="lzappgw-blue"
+    },
+    "aks_green" = {
+      name_prefix="green"
+      aks_turn_on=false
+      k8s_version="1.23.5"
+      appgw_name="lzappgw-green"
+    }
+  }
+}
+
+```
+
+After the deployment if the Landing Zone, than is possible to install a test application. The sample application to use is stored in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\07-AKS-cluster\sample-workload-for-agic-test.yaml".
+
+To deploy the sample application follow the same steps described at [Workload](./08-workload.md), the main difference is to apply the sample yaml to deploy the app.
+
```

Comment:
The steps in [Workload](./08-workload.md) are way overcomplicated and need to be simplified. Can't you remove all dependency on those steps given that you are using your own sample app?

Reply:
yes make sense, I have added to be in line and consistent with the standard guide. But yes, also for me the app that is mentioned in the standard guide seems really complex to deploy :)